src/server_manager/web_app/ui_components/account-list-app.ts

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
I'm wondering if we need this app. Isn't the top-level app the "account list app"?
The thing is that this app won't be able to start the server creation app (it runs on a sibling).
It's probably more useful if the app managing the accounts can also start the server creation. Otherwise it won't be doing much. It may still be worth it to have a list element though.